% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={R\_Assignment},
  pdfauthor={Cecilia Hodson},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[margin=1in]{geometry}
\usepackage{color}
\usepackage{fancyvrb}
\newcommand{\VerbBar}{|}
\newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\usepackage{framed}
\definecolor{shadecolor}{RGB}{248,248,248}
\newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
\newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\BuiltInTok}[1]{#1}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
\newcommand{\ExtensionTok}[1]{#1}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ImportTok}[1]{#1}
\newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\NormalTok}[1]{#1}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
\newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\RegionMarkerTok}[1]{#1}
\newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering

\title{R\_Assignment}
\author{Cecilia Hodson}
\date{2021-03-14}

\begin{document}
\maketitle

\hypertarget{part-i-data-inspection}{%
\section{Part I: Data Inspection}\label{part-i-data-inspection}}

\hypertarget{looking-at-the-files}{%
\subsubsection{Looking at the files:}\label{looking-at-the-files}}

\begin{Shaded}
\begin{Highlighting}[]
\KeywordTok{file.size}\NormalTok{(}\StringTok{"fang_et_al_genotypes.txt"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## [1] 11051939
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\KeywordTok{file.size}\NormalTok{(}\StringTok{"snp_position.txt"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## [1] 82763
\end{verbatim}

\hypertarget{creating-data-frames-for-.txt-files}{%
\subsubsection{Creating data frames for .txt
files:}\label{creating-data-frames-for-.txt-files}}

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{fang_genotypes <-}\StringTok{ }\KeywordTok{read.delim}\NormalTok{(}\StringTok{"fang_et_al_genotypes.txt"}\NormalTok{)}
\NormalTok{snp_positions <-}\StringTok{ }\KeywordTok{read.delim}\NormalTok{(}\StringTok{"snp_position.txt"}\NormalTok{)}
\end{Highlighting}
\end{Shaded}

\hypertarget{looking-at-the-dataframes}{%
\subsubsection{Looking at the
dataframes:}\label{looking-at-the-dataframes}}

\begin{Shaded}
\begin{Highlighting}[]
\KeywordTok{nrow}\NormalTok{(fang_genotypes)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## [1] 2782
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\KeywordTok{ncol}\NormalTok{(fang_genotypes)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## [1] 986
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\KeywordTok{nrow}\NormalTok{(snp_positions)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## [1] 983
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\KeywordTok{ncol}\NormalTok{(snp_positions)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## [1] 15
\end{verbatim}

\hypertarget{data-processing}{%
\subsubsection{Data Processing}\label{data-processing}}

Pulling genotype groups:

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{maize <-}\StringTok{ }\NormalTok{fang_genotypes[fang_genotypes}\OperatorTok{$}\NormalTok{Group }\OperatorTok{==}\StringTok{"ZMMIL"} \OperatorTok{|}\StringTok{ }\NormalTok{fang_genotypes}\OperatorTok{$}\NormalTok{Group }\OperatorTok{==}\StringTok{ "ZMMMR"} \OperatorTok{|}\StringTok{ }\NormalTok{fang_genotypes}\OperatorTok{$}\NormalTok{Group }\OperatorTok{==}\StringTok{ "ZMMLR"}\NormalTok{,]}
\NormalTok{teosinte <-}\StringTok{ }\NormalTok{fang_genotypes[fang_genotypes}\OperatorTok{$}\NormalTok{Group }\OperatorTok{==}\StringTok{ "ZMPIL"} \OperatorTok{|}\StringTok{ }\NormalTok{fang_genotypes}\OperatorTok{$}\NormalTok{Group }\OperatorTok{==}\StringTok{ "ZMPJA"} \OperatorTok{|}\StringTok{ }\NormalTok{fang_genotypes}\OperatorTok{$}\NormalTok{Group }\OperatorTok{==}\StringTok{"ZMPBA"}\NormalTok{,]}
\end{Highlighting}
\end{Shaded}

Transposing genotypes:

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{transposed_maize <-}\StringTok{ }\KeywordTok{as.data.frame}\NormalTok{(}\KeywordTok{t}\NormalTok{(maize[,}\OperatorTok{-}\DecValTok{1}\NormalTok{]))}
\KeywordTok{colnames}\NormalTok{(transposed_maize) <-}\StringTok{ }\NormalTok{maize}\OperatorTok{$}\NormalTok{Sample_ID}
\NormalTok{transposed_teosinte <-}\StringTok{ }\KeywordTok{as.data.frame}\NormalTok{(}\KeywordTok{t}\NormalTok{(teosinte[,}\OperatorTok{-}\DecValTok{1}\NormalTok{]))}
\KeywordTok{colnames}\NormalTok{(transposed_teosinte) <-}\StringTok{ }\NormalTok{teosinte}\OperatorTok{$}\NormalTok{Sample_ID}
\end{Highlighting}
\end{Shaded}

Pulling SNP Colums:

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{snp_cols <-}\StringTok{ }\NormalTok{snp_positions[, }\KeywordTok{c}\NormalTok{(}\DecValTok{1}\NormalTok{, }\DecValTok{3}\NormalTok{, }\DecValTok{4}\NormalTok{)]}
\end{Highlighting}
\end{Shaded}

Merging Files:

\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{#initial merging}
\NormalTok{merged_maize <-}\StringTok{ }\KeywordTok{merge}\NormalTok{(}\DataTypeTok{x =}\NormalTok{ snp_cols, }\DataTypeTok{y =}\NormalTok{ transposed_maize, }\DataTypeTok{by.x =} \StringTok{"SNP_ID"}\NormalTok{, }\DataTypeTok{by.y =} \StringTok{"row.names"}\NormalTok{, }\DataTypeTok{all.y =} \OtherTok{TRUE}\NormalTok{)}
\NormalTok{merged_teosinte <-}\StringTok{  }\KeywordTok{merge}\NormalTok{(}\DataTypeTok{x =}\NormalTok{ snp_cols, }\DataTypeTok{y=}\NormalTok{ transposed_teosinte, }\DataTypeTok{by.x =} \StringTok{"SNP_ID"}\NormalTok{, }\DataTypeTok{by.y =} \StringTok{"row.names"}\NormalTok{, }\DataTypeTok{all.y =} \OtherTok{TRUE}\NormalTok{)}

\CommentTok{#remove certain rows that are not data (i.e. "Group" and whatever "JG-OTU" is...)}
\NormalTok{merged_maize <-}\StringTok{ }\NormalTok{merged_maize[}\OperatorTok{-}\KeywordTok{c}\NormalTok{(}\DecValTok{14}\NormalTok{, }\DecValTok{16}\NormalTok{), ]}
\NormalTok{merged_teosinte <-}\StringTok{ }\NormalTok{merged_teosinte[}\OperatorTok{-}\KeywordTok{c}\NormalTok{(}\DecValTok{14}\NormalTok{, }\DecValTok{16}\NormalTok{), ]}

\CommentTok{#removing undesirable values(i.e., "multiple" & "unknown")}
\NormalTok{merged_maize <-}\StringTok{ }\NormalTok{merged_maize[}\OperatorTok{!}\NormalTok{(merged_maize}\OperatorTok{$}\NormalTok{Chromosome }\OperatorTok{==}\StringTok{ "multiple"}\NormalTok{), ]}
\NormalTok{merged_maize <-}\StringTok{ }\NormalTok{merged_maize[}\OperatorTok{!}\NormalTok{(merged_maize}\OperatorTok{$}\NormalTok{Chromosome }\OperatorTok{==}\StringTok{ "unknown"}\NormalTok{), ]}
\NormalTok{merged_maize <-}\StringTok{ }\NormalTok{merged_maize[}\OperatorTok{!}\NormalTok{(merged_maize}\OperatorTok{$}\NormalTok{Position }\OperatorTok{==}\StringTok{ "unknown"}\NormalTok{), ]}

\NormalTok{merged_teosinte <-}\StringTok{ }\NormalTok{merged_teosinte[}\OperatorTok{!}\NormalTok{(merged_teosinte}\OperatorTok{$}\NormalTok{Chromosome }\OperatorTok{==}\StringTok{ "multiple"}\NormalTok{), ]}
\NormalTok{merged_teosinte <-}\StringTok{ }\NormalTok{merged_teosinte[}\OperatorTok{!}\NormalTok{(merged_teosinte}\OperatorTok{$}\NormalTok{Chromosome }\OperatorTok{==}\StringTok{ "unknown"}\NormalTok{), ]}
\NormalTok{merged_teosinte <-}\StringTok{ }\NormalTok{merged_teosinte[}\OperatorTok{!}\NormalTok{(merged_teosinte}\OperatorTok{$}\NormalTok{Position }\OperatorTok{==}\StringTok{ "unknown"}\NormalTok{), ]}

\CommentTok{#sorting position column}
\NormalTok{merged_maize}\OperatorTok{$}\NormalTok{Position <-}\StringTok{ }\KeywordTok{as.numeric}\NormalTok{(merged_maize}\OperatorTok{$}\NormalTok{Position)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## Warning: NAs introduced by coercion
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{merged_teosinte}\OperatorTok{$}\NormalTok{Position <-}\StringTok{ }\KeywordTok{as.numeric}\NormalTok{(merged_teosinte}\OperatorTok{$}\NormalTok{Position)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## Warning: NAs introduced by coercion
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{merged_maize <-}\StringTok{ }\NormalTok{merged_maize[}\KeywordTok{order}\NormalTok{(merged_maize}\OperatorTok{$}\NormalTok{Position), ]}
\NormalTok{merged_teosinte <-}\StringTok{ }\NormalTok{merged_teosinte[}\KeywordTok{order}\NormalTok{(merged_teosinte}\OperatorTok{$}\NormalTok{Position), ]}
\end{Highlighting}
\end{Shaded}

Sorting by Chromosome Number:

\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{#for maize}
\NormalTok{maize_split_list <-}\StringTok{ }\KeywordTok{split}\NormalTok{(merged_maize, merged_maize}\OperatorTok{$}\NormalTok{Chromosome, }\DataTypeTok{drop =} \OtherTok{FALSE}\NormalTok{)}

\NormalTok{maize_chr_}\DecValTok{1}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'1'}
\NormalTok{maize_chr_}\DecValTok{2}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'2'}
\NormalTok{maize_chr_}\DecValTok{3}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'3'}
\NormalTok{maize_chr_}\DecValTok{4}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'4'}
\NormalTok{maize_chr_}\DecValTok{5}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'5'}
\NormalTok{maize_chr_}\DecValTok{6}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'6'}
\NormalTok{maize_chr_}\DecValTok{7}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'7'}
\NormalTok{maize_chr_}\DecValTok{8}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'8'}
\NormalTok{maize_chr_}\DecValTok{9}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'9'}
\NormalTok{maize_chr_}\DecValTok{10}\NormalTok{ <-}\StringTok{ }\NormalTok{maize_split_list}\OperatorTok{$}\StringTok{'10'}


\CommentTok{#for teosinte}
\NormalTok{teo_split_list <-}\StringTok{ }\KeywordTok{split}\NormalTok{(merged_teosinte, merged_teosinte}\OperatorTok{$}\NormalTok{Chromosome, }\DataTypeTok{drop =} \OtherTok{FALSE}\NormalTok{)}

\NormalTok{teo_chr_}\DecValTok{1}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'1'}
\NormalTok{teo_chr_}\DecValTok{2}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'2'}
\NormalTok{teo_chr_}\DecValTok{3}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'3'}
\NormalTok{teo_chr_}\DecValTok{4}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'4'}
\NormalTok{teo_chr_}\DecValTok{5}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'5'}
\NormalTok{teo_chr_}\DecValTok{6}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'6'}
\NormalTok{teo_chr_}\DecValTok{7}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'7'}
\NormalTok{teo_chr_}\DecValTok{8}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'8'}
\NormalTok{teo_chr_}\DecValTok{9}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'9'}
\NormalTok{teo_chr_}\DecValTok{10}\NormalTok{ <-}\StringTok{ }\NormalTok{teo_split_list}\OperatorTok{$}\StringTok{'10'}
\end{Highlighting}
\end{Shaded}

\hypertarget{part-ii-visualizing-data}{%
\section{Part II: Visualizing Data}\label{part-ii-visualizing-data}}

Install required packages:

\begin{Shaded}
\begin{Highlighting}[]
\KeywordTok{library}\NormalTok{(tidyverse)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## -- Attaching packages --------------------------------------- tidyverse 1.3.0 --
\end{verbatim}

\begin{verbatim}
## v ggplot2 3.3.3     v purrr   0.3.4
## v tibble  3.1.0     v dplyr   1.0.5
## v tidyr   1.1.3     v stringr 1.4.0
## v readr   1.4.0     v forcats 0.5.1
\end{verbatim}

\begin{verbatim}
## -- Conflicts ------------------------------------------ tidyverse_conflicts() --
## x dplyr::filter() masks stats::filter()
## x dplyr::lag()    masks stats::lag()
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\KeywordTok{library}\NormalTok{(tidyr)}
\KeywordTok{library}\NormalTok{(ggplot2)}
\KeywordTok{library}\NormalTok{(reshape2)}
\end{Highlighting}
\end{Shaded}

\begin{verbatim}
## 
## Attaching package: 'reshape2'
\end{verbatim}

\begin{verbatim}
## The following object is masked from 'package:tidyr':
## 
##     smiths
\end{verbatim}

SNPs Per Chromosome:

\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{#First, create a merged genotypes file:}
\NormalTok{genotypes <-}\StringTok{ }\KeywordTok{as.data.frame}\NormalTok{(}\KeywordTok{t}\NormalTok{(fang_genotypes[, }\DecValTok{-1}\NormalTok{]))}
\NormalTok{merged_genotypes <-}\StringTok{ }\KeywordTok{merge}\NormalTok{(}\DataTypeTok{x =}\NormalTok{ snp_cols, }\DataTypeTok{y =}\NormalTok{ genotypes, }\DataTypeTok{by.x =} \StringTok{"SNP_ID"}\NormalTok{, }\DataTypeTok{by.y =} \StringTok{"row.names"}\NormalTok{, }\DataTypeTok{all.y =} \OtherTok{TRUE}\NormalTok{)}
\NormalTok{merged_genotypes <-}\StringTok{ }\NormalTok{merged_genotypes[}\OperatorTok{-}\KeywordTok{c}\NormalTok{(}\DecValTok{14}\NormalTok{, }\DecValTok{16}\NormalTok{), ]}

\CommentTok{#Making the plot}
\KeywordTok{ggplot}\NormalTok{(merged_genotypes, }\KeywordTok{aes}\NormalTok{((Chromosome))) }\OperatorTok{+}\StringTok{ }\KeywordTok{geom_bar}\NormalTok{(}\DataTypeTok{stat=}\NormalTok{) }\OperatorTok{+}\StringTok{ }\KeywordTok{ggtitle}\NormalTok{(}\StringTok{"Number of SNP on each chromosome"}\NormalTok{) }\OperatorTok{+}
\StringTok{     }\KeywordTok{theme}\NormalTok{(}
         \DataTypeTok{axis.text =} \KeywordTok{element_text}\NormalTok{(}\DataTypeTok{colour =} \StringTok{"black"}\NormalTok{),}
         \DataTypeTok{axis.title.x =} \KeywordTok{element_text}\NormalTok{(}\DataTypeTok{colour =} \StringTok{"black"}\NormalTok{, }\DataTypeTok{size=}\KeywordTok{rel}\NormalTok{(}\DecValTok{1}\NormalTok{)),}
         \DataTypeTok{axis.title.y =} \KeywordTok{element_text}\NormalTok{(}\DataTypeTok{colour =} \StringTok{"black"}\NormalTok{, }\DataTypeTok{angle=}\DecValTok{90}\NormalTok{, }\DataTypeTok{size=}\KeywordTok{rel}\NormalTok{(}\DecValTok{1}\NormalTok{)),}
         \DataTypeTok{panel.background =} \KeywordTok{element_rect}\NormalTok{(}\DataTypeTok{fill=}\StringTok{"white"}\NormalTok{),}
         \DataTypeTok{panel.grid.minor.y =} \KeywordTok{element_line}\NormalTok{(}\DataTypeTok{size=}\DecValTok{3}\NormalTok{),}
         \DataTypeTok{panel.grid.major =} \KeywordTok{element_line}\NormalTok{(}\DataTypeTok{colour =} \StringTok{"black"}\NormalTok{),}
         \DataTypeTok{plot.background =} \KeywordTok{element_rect}\NormalTok{(}\DataTypeTok{fill=}\StringTok{"gray"}\NormalTok{))  }
\end{Highlighting}
\end{Shaded}

\includegraphics{R_README_files/figure-latex/unnamed-chunk-10-1.pdf}

\end{document}
